sktime/regression/compose/_multiplexer.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python3 -u
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements regressor for selecting among different model classes."""
+# based on MultiplexForecaster
+
+from sktime.base import _HeterogenousMetaEstimator
+from sktime.datatypes import MTYPE_LIST_PANEL, MTYPE_LIST_TABLE
+from sktime.regression._delegate import _DelegatedRegressor
+from sktime.regression.base import BaseRegressor
+
+__author__ = ["ksharma6"]
+__all__ = ["MultiplexRegressor"]
+
+
+class MultiplexRegressor(_HeterogenousMetaEstimator, _DelegatedRegressor):
+    """MultiplexRegressor for selecting among different models.
+
+    MultiplexRegressor facilitates a framework for performing
+    model selection process over different model classes.
+    It should be used in conjunction with GridSearchCV to get full utilization.
```

Comment:
This still doesn't exist, and you are planning to add that in the other PR, correct?

Reply:
Correct, would it be better to add a docstring example in this PR (using sklearn's grid search) and then update it after  [PR](https://github.com/sktime/sktime/pull/6118)  is merged (using an sktime implementation)?

Reply:
If it's compatible, sure.